homework/PavelBo/dataTypes/wrappers/src/test/java/com/github/pavelbopavel/WrapperDemoTest.java

```diff
@@ -0,0 +1,41 @@
+package com.github.pavelbopavel;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class WrapperDemoTest {
+    Logger log;
+
+    @Before
+    public void setUp(){
+        log = LoggerFactory.getLogger(WrapperDemoTest.class);
+    }
+    @Test
+    public void immutableDemo(){
+        Integer i1 = 10;
+        Integer i2 = i1;
+        Assert.assertEquals(i1.hashCode(), i2.hashCode());
```

Comment:
that is not how immutability tests. 
When you test if object hasn't been mutate, you should prove that it's original value did not changed. Revise your test